install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,79 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
```

Comment:
@tpoitras @bfallonf
Thanks for the review.

- grammar
The IBM Style Guide (section "Hyphens with compound words", p.54) says: "Do not use a hyphen after an adverb that ends in *-ly* because such terms are never ambiguous."  I think "dynamically provisioned storage" falls into this category.

- "in a storage-specific way"
This was my way to avoid saying "based on storage type", which is influenced by my dread of the word "type" -- it's an extremely overloaded term IME.  However, i now realize my experience is not relevant here.  The accepted term in the rest of the docs is indeed "storage type" so i'll stick w/ the program and save that battle for another day.

Anyway, the intent of this particular sentence was to rationalize the presentation of the analogous terms specific to each storage type (the table).  I would like to avoid that table being a surprise to the reader, or being viewed as superfluous.  Hmm, maybe i should add these musings as a comment in the source...